### DISCUSSION TITLE
Send Text Message Api Not delivering Message to destination-phone-number

### DISCUSSION THREAD

**Discussion by shoyshai** [netflie/whatsapp-cloud-api#181] [Q&A]
Dear Team.,
I am using the first basic example of 
$whatsapp_cloud_api->sendTextMessage('destination-phone-number', 'Hey there! I\'m using WhatsApp Cloud API. Visit https://www.netflie.es'); 
API is called successfully and the resposne received from API is  [body:protected] => {"messaging_product":"whatsapp","contacts":[{"input":"9975840786","wa_id":"919975840786"}],"messages":[{"id":"wamid.HBgMOTE5OTc1ODQwNzg2FQIAERgSMEYxMjc4QzZGMkQzRTY4RDI3AA=="}]} but it does not shows the   [message_status] => accepted , and message is not delivered to destination mobile. Please help as i just started exploring whatsapp api
![Screenshot_7](https://github.com/netflie/whatsapp-cloud-api/assets/71006096/789e8303-7f23-4c0c-b29e-90379141036e)

**Comment by aalbarca** [comment]
The first message must be a **TEMPLATE** message. Once the user answers then you can start sending other types of messages. 

**This is a WhatsApp requirement.**

**Reply by sebimartinus** [reply]
Do you know maybe why I can't send TEMPLATE messages created by me? That one created automatically by Facebook is working fine but those created by me not working...strange thing is that it works for 2 months but since 2 weeks ago not working anymore...Thanks!
